ReceiveMessage/Scripts/sendMessage.ts

```diff
@@ -4,17 +4,17 @@ import fixedMessages from '../fixedMessages'
 
 import { IUserState } from '../../models/UserState'
 
-const formResponse = async function (userState : IUserState, message : string) : Promise<IMessage>{
+const formResponse = async function (userState: IUserState, message: string): Promise<IMessage> {
     const currMessageId = userState.currMessage
     if (currMessageId == null) {
-        userState.currMessage = fixedMessages.get('welcome')
+        userState.currMessage = await fixedMessages.get('welcome')
```

Comment:
One small thing. userState.currMessage expects **an id of a chatbot message**, while the fixed message is the chatbot message itself. This should be = (await fixedMessages.get('welcome'))["_id"]